python/tests/table_html/test_to_html_dataset.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+import pytest
+from bs4 import BeautifulSoup
```

Comment:
Does this need to be installed on Travis?

Reply:
Yes. I thought I had it to `requirements.txt` but I can't see it, so I'll do that